src/Searchable/Aggregator.php

```diff
@@ -184,9 +186,11 @@ public static function removeAllFromSearch(): void
         foreach ((new static)->getModels() as $model) {
             $instance = new $model;
 
-            $instance->newQuery()->orderBy($instance->getKeyName())->get()->map(function ($model) {
-                return static::create($model);
-            })->unsearchable();
+            $instance->newQuery()->orderBy($instance->getKeyName())->chunk(config('scout.chunk.searchable', 500), function ($models) {
```

Comment:
Since we now require scout `6.0` I just have removed this method in order to use the scout one: https://github.com/algolia/scout-extended/commit/018e2dd0928435e533b7c3f615ccb6e4aa7e8cb3